TombEngine/Objects/TR3/Entity/tr3_flamethrower.cpp

```diff
@@ -20,21 +20,37 @@ using namespace TEN::Math::Random;
 
 namespace TEN::Entities::Creatures::TR3
 {
+	constexpr auto FLAMETHROWER_ATTACK_RANGE = SQUARE(SECTOR(1.5f));
+	constexpr auto FLAMETHROWER_IDLE_RANGE	 = SQUARE(SECTOR(2));
+	constexpr auto FLAMETHROWER_AWARE_RANGE	 = SQUARE(SECTOR(8));
+
+	constexpr auto FlamethrowerWalkAngle = ANGLE(5.0f);
```

Comment:
Why this constexpr is in PascalCase and not in ALL_CAPS?

Reply:
I missed this one while reformatting all the others from PascalCase. Will fix.